scripts/e2e.sh

```diff
@@ -1,13 +1,7 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
```

Comment:
Is this intentional? Errors won’t fail the script.

Reply:
yeah, it is intentional (github action default has that for bash script)

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrunshell

![image](https://github.com/runatlantis/atlantis/assets/1580956/4567c288-6888-493f-bd0e-9bf41594d5d6)

Reply:
Right, that is for the commands you specify on the `run` key. This is a bash file, afaik it will start a new bash process that doesn’t inherit those configurations. For it to be true the file would have to be sourced not executed, then it would run on the same process.

Reply:
I could not remember what exactly happened before, but okay with bringing it back.